mne_bids/stats.py

```diff
@@ -0,0 +1,78 @@
+"""Some functions to extract stats from a BIDS dataset."""
+
+# Authors: Alex Gramfort <alexandre.gramfort@inria.fr>
+#
+# License: BSD (3-clause)
+
+
+from mne_bids import BIDSPath
+
+
+def count_events(root, subject=None, session=None, task=None, run=None):
+    """Count events
```

Comment:
I feel this should accept a `BIDSPath` – or even be exposed as a `BIDSPath` method?

Reply:
done